pkg/bundle/bundle.go

```diff
@@ -136,22 +142,33 @@ func (b *ProtobufBundle) VerificationContent() (verify.VerificationContent, erro
 
 	switch content := b.VerificationMaterial.GetContent().(type) {
 	case *protobundle.VerificationMaterial_X509CertificateChain:
-		certs := content.X509CertificateChain.GetCertificates()
-		certificates := make([]*x509.Certificate, len(certs))
-		var err error
-		for i, cert := range content.X509CertificateChain.GetCertificates() {
-			certificates[i], err = x509.ParseCertificate(cert.RawBytes)
+		var parsedCert *x509.Certificate
+		for i, eachCert := range content.X509CertificateChain.GetCertificates() {
+			thisCert, err := x509.ParseCertificate(eachCert.RawBytes)
 			if err != nil {
 				return nil, ErrValidationError(err)
 			}
+
+			if i == 0 {
+				parsedCert = thisCert
+			}
 		}
-		if len(certificates) == 0 {
+		if parsedCert == nil {
 			return nil, ErrMissingVerificationMaterial
 		}
-		certChain := &CertificateChain{
-			Certificates: certificates,
+		cert := &Certificate{
```

Comment:
We should keep the previous logic to copy the whole chain. That supports private deployments that distribute intermediates too.

As a follow up, we should check that no self-signed root is present in the certificate chain.

Reply:
Per below, we'll track this on https://github.com/sigstore/sigstore-go/issues/132